workshop/exercise-1/README.md

```diff
@@ -34,40 +29,46 @@ You must already have a [cluster created](https://console.bluemix.net/docs/conta
     4. uk-south
     5. us-east
     6. us-south
-    Enter a number> 5
+    Enter a number> 6
     ```
-    
+
 6. Install the Kubernetes CLI. Go to the [Kubernetes page](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl) to install the CLI and follow the steps.
 
 
+> To proceed, your Kubernetes cluster must be started - the state should change from "Pending" to "Normal". Here are the steps to access your cluster. If your cluster is taking too long to create, ask a room moderator for help.
```

Comment:
"room moderator" is kind of odd for a self-guided thing. Can we make this more generic, or have a section neat the top of the workshop that talks about how to get help?  I think listing the ibm-cloud slack channel might be nice for those off-line people.  I'm not sure we need to say "ask the moderator" if this is a live class since that should be pretty obvious.